src/explore-education-statistics-common/src/components/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import { ArrowLeft, ArrowRight } from '@common/components/ArrowIcons';
+import generatePageNumbers from '@common/components/util/generatePageNumbers';
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+const paginationLinkClassName = 'govuk-link govuk-pagination__link';
+
+interface LinkRenderProps {
+  'aria-current'?: 'page' | undefined;
+  'aria-label'?: string;
+  children: ReactNode;
+  className: string;
+  rel?: 'next' | 'prev';
+  to: string;
+}
+
+export interface PaginationProps {
+  baseUrl: string;
+  currentPage: number;
+  label?: string;
+  queryParams?: Record<string, unknown>;
+  renderLink: (props: LinkRenderProps) => ReactNode;
+  totalPages: number;
+}
+
+const Pagination = ({
+  baseUrl,
+  currentPage,
+  label = 'Pagination',
+  queryParams,
+  renderLink,
+  totalPages,
+}: PaginationProps) => {
+  const pageNumbers = generatePageNumbers({
+    currentPage,
+    totalPages,
+  });
+
+  if (!pageNumbers.length) {
+    return null;
+  }
+
+  const queryString = queryParams
+    ? `&${Object.keys(queryParams)
+        .map(key => `${key}=${queryParams[key]}`)
+        .join('&')}`
+    : '';
```

Comment:
This is too simplistic for creating a query string as it doesn't handle URL encoding. We should use the `qs` to handle this instead. In fact, there is already an `appendQuery` function that uses this in the admin project (it also handles URLs that may already contain a query string).

Would suggest moving this into common and using it here:
```ts
to: appendQuery(baseUrl, {
  ...queryParams,
  page: currentPage - 1
})
```